ios/FluentUI/Date Time Pickers/Date Picker/DatePickerController.swift

```diff
@@ -114,15 +117,16 @@ class DatePickerController: UIViewController, GenericDateTimePicker {
         self.mode = mode
         self.leftBarButtonItem = leftBarButtonItem
         self.rightBarButtonItem = rightBarButtonItem
+        self.calendarConfiguration = calendarConfiguration ?? CalendarConfiguration.default
 
         super.init(nibName: nil, bundle: nil)
 
         defer {
             self.startDate = startDate
             self.endDate = endDate
         }
-
-        calendarViewDataSource = CalendarViewDataSource(styleDataSource: self)
+        
+        calendarViewDataSource = CalendarViewDataSource(styleDataSource: self, calendarConfiguration: self.calendarConfiguration)
```

Comment:
will fix the spaces

Reply:
(Having invisibles on should help with this if you don't have that on already!)